crates/ra_db/src/input.rs

```diff
@@ -101,6 +102,10 @@ impl CrateName {
     pub fn normalize_dashes(name: &str) -> CrateName {
         Self(SmolStr::new(name.replace('-', "_")))
     }
+
+    pub fn get_name(&self) -> String {
```

Comment:
Let's do `impl std::fmt::Display for CrateName` instead.

Reply:
In general, it's a good rule of thumb to think, every time you add a `-> String` function, "would Display work?".